src/Categories/Bicategory.agda

```diff
@@ -63,6 +63,29 @@ record Bicategory o ℓ e t : Set (suc (o ⊔ ℓ ⊔ e ⊔ t)) where
   _▷_ : {A B C : Obj} {f g : A ⇒₁ B} (h : B ⇒₁ C) (α : f ⇒₂ g) → h ∘ₕ f ⇒₂ h ∘ₕ g
   _ ▷ α = id₂ ⊚₁ α
 
+  -- Coherence morphisms
```

Comment:
There are variants of these in [Bicategory.Extras](https://github.com/agda/agda-categories/blob/1ba82d98faddc750fe4c0363af0d1e21292a9085/src/Categories/Bicategory/Extras.agda#L38), including shorthands. Probably not a good idea to duplicate them.